modules/gdnative/nativescript/api_generator.cpp

```diff
@@ -110,7 +110,7 @@ struct ClassAPI {
 	bool is_singleton;
 	bool is_instanciable;
 	// @Unclear
-	bool is_creatable;
+	//bool is_creatable;
```

Comment:
@karroffel Would that break the API?

Reply:
No, this is fine and can be removed, it's only used in a line of code that's commented out

Reply:
Thanks for the review. These lines are now removed.